modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectValidationRuleResourceImpl.java

```diff
@@ -333,6 +333,27 @@ private ObjectValidationRule _toObjectValidationRule(
 					return serviceBuilderObjectValidationRuleSetting;
 				}
 
+				if (StringUtil.equals(
```

Comment:
Can we create a private method to reuse this logic? maybe something like replace or transform property name
So you would just need to pass two Strings, for example: ercPropertyName, idPropertyName
case 1: `NAME_OUTPUT_OBJECT_FIELD_EXTERNAL_REFERENCE_CODE, NAME_OUTPUT_OBJECT_FIELD_ID`
case 2: `NAME_KEY_OBJECT_FIELD_EXTERNAL_REFERENCE_CODE, NAME_KEY_OBJECT_FIELD_ID`